code/src/reusables/urls.js

```diff
@@ -0,0 +1,2 @@
+const api_key='d54701f7b694b0188ac7819c5b24df16';
+export const MOVIES_URL = `https://api.themoviedb.org/3/movie/popular?api_key=${api_key}&language=en-US&page=1`;
```

Comment:
A separate variable for the api key is a great idea! We have clever foxes in our foxes team! These two fox-mates are proud of you! 🦊🦊